pkg/portal/appresource/manager_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	apimodel "github.com/authgear/authgear-server/pkg/api/model"
 	"github.com/authgear/authgear-server/pkg/lib/config"
 	configtest "github.com/authgear/authgear-server/pkg/lib/config/test"
+	"github.com/authgear/authgear-server/pkg/lib/web"
```

Comment:
During development, I found that actually one of the package imports performed a side effect that register `AuthgearYAMLDescriptor` into `resource.DefaultRegistry` so that the manager was able to find `authgear.yaml` in the test cases.

While other static resources cannot be found in the test cases because we didn't import `github.com/authgear/authgear-server/pkg/lib/web` and those static resources weren't registered into default registry.